pysat/tests/test_constellation.py

```diff
@@ -59,16 +59,6 @@ def test_str(self):
             "\npysat Constellation object:\ntesting\ntesting\n"
 
 
-class TestAdditionIdenticalInstruments:
-    def setup(self):
-        self.const1 = pysat.Constellation(name='testing')
-        self.const2 = pysat.Constellation(name='single_test')
-
-    def teardown(self):
-        del self.const1
-        del self.const2
-
-
```

Comment:
These are also not being run.  Leftovers from the removal of the addition methods for Constellations.